README.md

```diff
@@ -59,42 +59,18 @@ You have to maintain the list of components in `demo/Demo.react.js`.
 
 ### Code quality and tests
 
-#### To run lint and unit tests:
+### To run integration tests (test_integration.py)
+We run our integration tests on CircleCI with help from Tox. There’s a tox.ini file which holds the configuration, refer to [tox's documentation](http://tox.readthedocs.io/en/latest/index.html) for help. I found that I needed to set environment variables in my terminal, like `TOX_PYTHON_27` to my version of python that I wanted tox to use. So running:
```

Comment:
This is super nit-picky 🙈 but I prefer when the READMEs use the same pro-noun scheme throughout. So instead of saying: 'I found I needed to...'
use: 'You may need to...'

Reply:
Good point! Although 'I' is used in other parts of the readme as well, as in 'I recommend installing ...' etc. I'll change it to 'you may need to' regardless!